Add StoreSections component.

justincan opened the pull request:
- Copied over from snapweb (StoreHeader).

earnubs commented on src/Store/StoreSections.js:
```diff
@@ -0,0 +1,38 @@
+import React from "react";
```
Hi! You can also use import to grab module members, which makes the code a little neater, like this:
```
import React, { Component, PropTypes } from "react";
...
export default class StoreSections extends Component {
  static propTypes = {
    title: PropTypes.string
  }
}
```

justincan replied:
I'm taking this as a comment - I'd rather leave it. It has all the usual (minimal) advantages and (larger) disadvantages of global importation. It's not a common idiom either.

earnubs replied:
Hi, can you explain a little more what you mean? What are the disadvantages? Which bit isn't common?

earnubs commented on src/index.js:
```diff
@@ -21,3 +21,4 @@ export { default as SnapPageLogin } from './SnapPage/SnapPageLogin'
 export { default as SnapPagePurchase } from './SnapPage/SnapPagePurchase'
 export { default as SnapPageSummary } from './SnapPage/SnapPageSummary'
 export { default as SnapPageTags } from './SnapPage/SnapPageTags'
+export { default as StoreSections } from './Store/StoreSections'
```
I think these can just be:
```
export StoreSections from './Store/StoreSections';
```

but maybe there's a reason for doing it this way?

justincan replied:
That seems fine, it's exporting the default export.

Commit:
Import Snapweb styles into toolkit.

- Add Vanilla CSS toolkit
- Add sass processing step
- Import snapweb styles from snapweb project.
- Exporting vanilla based snapweb styles globally

Commit:
Add StoreSections component.

- Copied over from snapweb (StoreHeader).
